views/partials/tournament_1v1_game_finished.ejs

```diff
@@ -0,0 +1,9 @@
+<div class="rowContainer" style="grid-column: 1 / 13;">
```

Comment:
Please take a look at https://github.com/DavidRisch/kicker/blob/master/html/match_row.html for the correct layout